index.html

```diff
@@ -9,126 +9,154 @@
-    <a href="https://www.linkedin.com/in/justo-rago-0714b5208/"><img src="./images/Normal-Button/Tertiary/Icons/Group.png" alt="LinkedIn"></a>
-    <a href="https://angel.co/"><img src="./images/Normal-Button3/Tertiary/Icons/Vector.png" alt="AngelList"></a>
-    <a href="https://twitter.com/JustoRago"><img src="./images/Normal-Button2/Tertiary/Icons/Vector.png" alt="Twitter"></a>
-    <a href="https://medium.com/"><img src="./images/Normal-Button1/Tertiary/Icons/Vector.png" alt="Medium"></a>
-  </section>
-  <section id="internal">
-    <a id="work-link" href="#works"><img src="./images/Group-55.png" alt="arrow">Work</a>
+    <div id="internal-external">
+      <a class="work-link browser" href="#works">Work<img src="./images/Group-55.png" alt="arrow"></a>
+    <ul class="footer-list" id="middlelist">
+      <li>
+        <a href="https://github.com/asdt560"><img src="./images/Normal-Button/Tertiary/Icons/Vector.png" alt="Github"></a>
+      </li>
+      <li>
+        <a href="https://www.linkedin.com/in/justo-rago-0714b5208/"><img src="./images/Normal-Button/Tertiary/Icons/Group.png" alt="LinkedIn"></a>
+      </li>
+      <li>
+        <a href="https://angel.co/"><img src="./images/Normal-Button3/Tertiary/Icons/Vector.png" alt="AngelList"></a>
+      </li>
+      <li>
+        <a href="https://twitter.com/JustoRago"><img src="./images/Normal-Button2/Tertiary/Icons/Vector.png" alt="Twitter"></a>
+      </li>
+      <li>
+        <a href="https://medium.com/"><img src="./images/Normal-Button1/Tertiary/Icons/Vector.png" alt="Medium"></a>
+      </li>
+    </ul>
+    <a class="work-link browser" href="#works"><img src="./images/Group-55.png" alt="arrow">Work</a>
+  </div>
+    <section id="internal">
+      <a class="work-link" href="#works"><img src="./images/Group-55.png" alt="arrow">Work</a>
+    </section>
   </section>
   <section id="works">
-    <h4>MY RECENT WORK</h4>
+    <h4>My recent work</h4>
     <span id="span3" class="divide"></span>
     <section id="c1" class="card">
       <img class="placeholder-image" src="./images/Img-Placeholder(1).png" alt="A Placeholder">
-      <h5 class="card-title">Multi-Post Stories</h5>
-      <p class="blurb2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
-      <ul class="card-list">
-        <li>Html</li>
-        <li>Css</li>
-        <li>Javascript</li>
-      </ul>
-      <button class="project-button">See Project</button>
+      <div class="card-text">
+        <h5 class="card-title">Multi-Post Stories</h5>
+        <p class="blurb2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <ul class="card-list">
```

Comment:
- [OPTIONAL] Please make sure your My recent work section is aligned on the center of the page when the width of the page is bigger than:  `@media screen and (min-width: 1080px)`